FAQ.md

```diff
@@ -0,0 +1,113 @@
+
+[[Back to main IRS README](README.md)]
```

Comment:
GitHub provided navigation within the project - this is not necessery IMHO.

Reply:
What do you mean by GitHub provided navigation in the Project? I like the reference to the parent document. This makes navigation in the browser as well as in the IDE easier.